src/app/GitUI/CommandsDialogs/SettingsDialog/ShellExtension/ShellExtensionManager.cs

```diff
@@ -42,7 +42,11 @@ public static bool IsRegistered()
         /// <exception cref="FileNotFoundException">If at least one necessary for registration file wasn't found</exception>
         /// <exception cref="Win32Exception">If user canceled elevation dialog (when ex.NativeErrorCode == 1223)</exception>
         /// <exception cref="Exception">Other potential error</exception>
-        public static void Register() => RunRegSvrForShellExtensionDlls("/s {0}");
+        public static void Register()
+        {
+            AppSettings.SetInstallDir(AppSettings.GetGitExtensionsDirectory());
```

Comment:
Wouldn't we need the same for other invocation of `RunRegSvrForShellExtensionDlls`?
Wouldn't it make sense to call `AppSettings.SetInstallDir` in `RunRegSvrForShellExtensionDlls`?

Reply:
> Wouldn't it make sense to call AppSettings.SetInstallDir in RunRegSvrForShellExtensionDlls?

Not for deregistration.

> Wouldn't we need the same for other invocation of RunRegSvrForShellExtensionDlls?

Deregistration is the only other caller.

Reply:
Makes sense